build/taskctl/tasks.yaml

```diff
@@ -132,29 +146,33 @@ tasks:
   # Infrastructure tasks
   infra:init:
     context: powershell
-    description: Initialise Terraform for Azure
+    description: Initialise Terraform for the environment
     command:
-      - Invoke-Terraform -Init -backend "$env:TF_BACKEND_ARGS" -Path $env:TF_FILE_LOCATION -Debug
-      - Invoke-Terraform -Workspace -Arguments $env:ENV_NAME -Path $env:TF_FILE_LOCATION -Debug
+      - ln -sfn /usr/local/terraform/1.9.3/bin /usr/local/terraform/bin;
+        Invoke-Terraform -Init -backend "$env:TF_BACKEND_ARGS" -Path $env:TF_FILE_LOCATION -Debug;
+        Invoke-Terraform -Workspace -Arguments $env:ENV_NAME -Path $env:TF_FILE_LOCATION -Debug
 
   infra:plan:
     context: powershell
     description: Plan Terraform
     command:
-      - Invoke-Terraform -Plan -Path $env:TF_FILE_LOCATION -Arguments "-input=false","-out=tfplan" -Debug
+      - ln -sfn /usr/local/terraform/1.9.3/bin /usr/local/terraform/bin;
```

Comment:
If you use `ensono/eir-infrastructure:1.1.214` then you can drop a file in the root of the repo `.terraform-version` with the version number you want, the container has `1.9.3`.